pulp_ansible/app/tasks/collections.py

```diff
@@ -460,8 +473,59 @@ def _collection_list_downloader(self, api_version, collection_endpoint, page_num
             collection_list_url = f"{collection_endpoint}?limit={page_size}&offset={offset}"
         return self.remote.get_downloader(url=collection_list_url)
 
+    async def _download_unpaginated_repo_metadata(self):
+        self._unpaginated_collection_metadata = None
+        self._unpaginated_collection_version_metadata = None
+
+        root_endpoint, api_version = await self._get_root_api(self.remote.url)
+        self._api_version = api_version
+        collection_endpoint = f"{root_endpoint}/metadata/collections/?deprecated=true"
```

Comment:
we can, do you have any suggestion? Naming is always hard for me

Reply:
@fao89 for this I think we don't want server-side filtering, we want all the data. For right now it may not make a difference because really we just need this endpoint to get the `deprecated=true` but over time we probably want the fetching of this endpoint to respond with all data, not filtered data.